x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -78,10 +79,22 @@ export interface PolicyListState {
   isLoading: boolean;
 }
 
+/**
+ * Policy list store state
+ */
+export interface PolicyDetailsState {
+  /** Array of policy items  */
```

Comment:
Is is supposed to be an array of `PolicyData` or a single one?

Reply:
a single one, you caught my c/p.  Corrected.